modules/cli/src/main/scala/scala/cli/commands/util/FmtUtil.scala

```diff
@@ -0,0 +1,202 @@
+package scala.cli.commands.util
+
+import com.typesafe.config.ConfigParseOptions
+import com.typesafe.config.ConfigSyntax
+import com.typesafe.config.parser.ConfigDocument
+import com.typesafe.config.parser.ConfigDocumentFactory
+import scala.build.Logger
+import scala.util.control.NonFatal
+
+object FmtUtil {
+  private def getGitRoot(workspace: os.Path, logger: Logger): Option[String] =
+    try {
+      val result = os.proc("git", "rev-parse", "--show-toplevel").call(
+        cwd = workspace,
+        stderr = null
```

Comment:
I'm not sure passing `null` is fine here. If you just want to discard stderr, maybe something like this should do:
```suggestion
        stderr = os.ProcessOutput.ReadBytes((_, _) => ())
```

But if that command doesn't print garbage by default, `os.Inherit` should do. Else `os.Pipe` could work, that should put the error output in the exception below, if the command fails.